api/VRDisplay.json

```diff
@@ -27,17 +27,30 @@
             "version_removed": "79"
           },
           "firefox": [
+            {
+              "version_added": "98",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.vr.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
             {
               "version_added": "55",
+              "version_removed": "98",
               "notes": "Windows support was enabled in Firefox 55."
             },
             {
               "version_added": "64",
+              "version_removed": "98",
               "notes": "macOS support was enabled in Firefox 64."
             }
           ],
           "firefox_android": {
-            "version_added": "55"
+            "version_added": "55",
+            "version_removed": "98"
```

Comment:
@ddbeck Not sure what to do for android. It is definitely not present from FF98 as it is hidden behind a preference. However my understanding is that it was only enabled by default for Windows/macOS, and only for nightly/dev on other platforms, including Android. 

My **guess** is that it was never supported on android so this should be set false. However it is possible it was initially supported and then moved to only being enabled in dev/nightly -  in which case we would need to find the specific version. 

Thoughts?  [Note, have only made this change here, pending discussion]

Reply:
Hmm. I'm not sure. Some poorly directed searching shows it was available in Nightly at some point, but I don't know whether it was ever enabled. I'm going to ask for some advice here:

@queengooborg have you ever tried any WebVR stuff out on a mobile browser? The collector doesn't cover it, but maybe you've got a recommendation on how to go about testing this? (I don't want or care to do a comprehensive test of the API, just a quick reality check on whether it was supported in Firefox for Android or not)

Reply:
FYI, As discussed in 1:1 I am going to roll it out like this everywhere on the assumption the original info is correct. This will  be better than we have now, going forward. If @queengooborg or someone else finds we're wrong we can adjust, just as we are doing now.

Reply:
Yes, that's fine. Seems to have only been applied in this one place, however. Want to push another commit that does this to all of Firefox for Android?

Reply:
On further review, it appears that the top-level interfaces have been updated, but not their descendants. This is the cause of the linter failure; the descendants are inconsistent with the parent features.

Reply:
Arrrg. Thanks. Fixed.